centreon/tests/e2e/features/Cloud-notifications/01-notification-create/index.ts

```diff
@@ -188,15 +195,17 @@ When('the user selects the {string}', (contactSettings: string) => {
 });
 
 When('the user defines a mail subject', () => {
-  cy.getByLabel({ label: 'Subject' }).type(
-    `{selectAll}{backspace}Subject notification for ${globalResourceType} and ${globalContactSettings}`
-  );
+  const subject = globalResourceType
+    ? `{selectAll}{backspace}Subject notification for ${globalResourceType} and ${globalContactSettings}`
+    : `Subject notification for 1000 services`;
+  cy.getByLabel({ label: 'Subject' }).clear().type(subject);
 });
 
 When('the user defines a mail body', () => {
-  cy.getByLabel({ label: 'EmailBody' }).type(
-    `{selectAll}{backspace}Body notification for ${globalResourceType} and ${globalContactSettings}`
-  );
+  const body = globalResourceType
+    ? `{selectAll}{backspace}Body notification for ${globalResourceType} and ${globalContactSettings}`
+    : `{selectAll}{backspace}Body notification for 1000 services`;
+  cy.getByLabel({ label: 'EmailBody' }).clear().type(body);
```

Comment:
no need to clear since you use `{selectAll}{backspace}`
```suggestion
  cy.getByLabel({ label: 'EmailBody' }).type(body);
```